server/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "server",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "start": "node ./bin/www"
+  },
+  "dependencies": {
+    "body-parser": "~1.15.1",
+    "cookie-parser": "~1.4.3",
+    "debug": "~2.2.0",
```

Comment:
Debug is needed to log whether the system system goes on the right way. I think it would be better to let it in dependency because as it is 'required' and used in .js files. The server will not run with this dependency.